test/passing/js_source.ml.ocp

```diff
@@ -95,11 +96,13 @@ let () =
   [%foo
     while () do
       ()
-    done [@foo]];
+    done
+    [@foo]];
   [%foo
     for x = () to () do
       ()
-    done [@foo]];
+    done
+               [@foo]];
```

Comment:
due to ocp-indent

Reply:
Before I merge this issue @ceastlund @hhugo is this kind of ocp-indent quirks an issue for JaneStreet? I didn't see any regression using test_branch so I guess this code pattern isn't that common

Reply:
We run into this sometimes and yeah, it's really annoying. But it's not super common.